### PR TITLE
Fix build and theia-plugin bin for windows run

### PR DESCRIPTION
**Pull request by amiramw**
Signed-off-by: Amiram Wingarten <amiramw@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "author": "Florent Benoit",
   "license": "EPL-2.0",
   "bin": {
-    "theia:plugin": "./dist/index.js"
+    "theia-plugin": "./dist/index.js"
```
[the remark this answers is not in the document]
Comment (benoitf) [reply package.json]: creating both would be more backward compliant yes

[the remark this answers is not in the document]
Comment (amiramw) [reply package.json]: Now I kept both of them.

File: package.json [hunk 2]
```diff
@@ -50,6 +51,7 @@
 
   },
   "jest": {
+    "testURL": "http://localhost/",
```
Comment (benoitf) [inline package.json]: hello, do we need it ?

Comment (amiramw) [reply package.json]: removed

**Comment by benoitf** [comment]
ok let's wait travis job and we can merge.
thanks !

**Comment by benoitf** [comment]
thanks @amiramw !

**Comment by amiramw** [comment]
@benoitf when should new version publish to npm?

**Comment by benoitf** [comment]
@amiramw  https://travis-ci.org/eclipse/theia-plugin-packager/builds/485947278

**Pull request merged**